package.json

```diff
@@ -12,17 +12,17 @@
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.19.2",
     "cors": "^2.8.5",
-    "create-components-react": "^2.1.4",
+    "create-components-react": "git+https://github.com/horondi/template_creator.git",
     "react": "^16.13.1",
     "react-customer-chat": "^1.0.2",
     "react-dom": "^16.13.1",
     "react-redux": "^7.2.0",
     "react-scripts": "3.4.1",
+    "react-scripts-config": "^1.0.17-13",
     "react-scroll-up-button": "^1.6.4",
     "react-star-ratings": "^2.3.0",
     "redux": "^4.0.5",
-    "redux-thunk": "^2.3.0",
-    "react-scripts-config": "^1.0.17-13"
+    "redux-thunk": "^2.3.0"
```

Comment:
Do we need `redux-thunk`? We use `redux-saga` instead

Reply:
redux-thunk deleted